de.dlr.sc.overtarget.language.ui/plugin.xml

```diff
@@ -13,6 +13,18 @@
 			name="Overtarget Editor">
 		</editor>
 	</extension>
+ <extension
+       point="org.eclipse.ui.editors">
+    <editor
```

Comment:
Why are we registering a new editor here? What do we need it for? Does that maybe belong to the other PR?

Reply:
Yes, I'll delete that, thank you 👍